Formula/gmt.rb

```diff
@@ -32,14 +33,26 @@ class Gmt < Formula
     sha256 "8d47402abcd7f54a0f711365cd022e4eaea7da324edac83611ca035ea443aad3"
   end
 
+  # The following two patches fix a problem in detecting locally installed
+  # html pages (https://github.com/GenericMappingTools/gmt/issues/1960).
+  # They must be removed when GMT 6.0.1 is released.
+  patch do
+    url "https://github.com/GenericMappingTools/gmt/commit/b65dc6ebe7eba396b57dbad59e85b7ba7e8e2908.patch?full_index=1"
+  end
+
+  patch do
```

Comment:
Use the patch from this commit instead of the pull request.

https://github.com/GenericMappingTools/gmt/commit/daf646554a25cf1a4a94255b694afc43028f9696.patch?full_index=1